lib/src/__tests__/DateTimePickerTestingLib.test.tsx

```diff
@@ -96,4 +96,20 @@ describe('<DateTimePicker />', () => {
     fireEvent.click(screen.getByLabelText('open next view'));
     expect(screen.getByLabelText('open next view')).toBeDisabled();
   });
+
+  it('allows to select the same day and move to the next view', () => {
+    const onChangeMock = jest.fn()
+    render(
+      <StaticDateTimePicker
+        onChange={onChangeMock}
+        renderInput={(props) => <TextField {...props} />}
+        value={utilsToUse.date('2018-01-01T00:00:00.000Z')}
+      />
+    );
+
+    fireEvent.click(screen.getByLabelText("Jan 1, 2018"))
+    expect(onChangeMock).toHaveBeenCalled()
```

Comment:
The test pass without the fix. Could you update it to be failing (otherwise we are testing something else)?

Reply:
It is failing for me.

Reply:
You are right, it does fail +1